### ISSUE TITLE
Time zone can't be saved in My Profile

### ISSUE THREAD

**Issue by Antoniano** [nilsteampassnet/TeamPass#1601]
### Steps to reproduce
1. Go to My Profile and Change Time Zone (Europe/Kiev).
2. Click green "checkbox" button - it save the timezone.
3. Log out and Log in again - Time zone isn't set anymore.

P.S. time zone wasn't defined on Settings tab in admin console.

### Expected behaviour
Time zone in My Profile must be saved.

### Actual behaviour
Upon next log in time zone is not set again.

### Server configuration
**Operating system**: Ubuntu 14.04

**Web server:** Apache2

**Database:** MySQL

**PHP version:** 5.5.9

**Teampass version:** 2.1.26

**Updated from an older Teampass or fresh install:** fresh install

### Client configuration
**Browser:** Mozilla

**Operating system:** Windows 10

#### Web server error log
```
Insert your webserver log here
```

#### Firebug log ([How to?](http://teampass.net/2014-02-09-how-to-communicate-an-error-log))
```
Insert the Firebug log here
```

**Comment by nilsteampassnet** [comment]
Corrected in 2.1.27

You can do the next manual change.

- open file `sources/identify.php`
- find and replace
```
            // get personal settings
            if (!isset($data['treeloadstrategy']) || empty($data['treeloadstrategy'])) $data['treeloadstrategy'] = "full";
```
by
```
            // get personal settings
            if (!isset($data['treeloadstrategy']) || empty($data['treeloadstrategy'])) $data['treeloadstrategy'] = "full";
            $_SESSION['user_settings']['usertimezone'] = $data['usertimezone'];
```

**Comment by nilsteampassnet** [comment]
Corrected
